step-framework-parent/step-framework-collections/src/test/java/step/core/collections/inmemory/InMemoryCollectionTest.java

```diff
@@ -1,13 +1,25 @@
 package step.core.collections.inmemory;
 
-import java.util.Properties;
-
+import org.junit.Assert;
+import org.junit.Test;
 import step.core.collections.AbstractCollectionTest;
+import step.core.collections.Document;
+
+import java.util.Properties;
 
 public class InMemoryCollectionTest extends AbstractCollectionTest {
 
-	public InMemoryCollectionTest() {
-		super(new InMemoryCollectionFactory(new Properties()));
-	}
+    public InMemoryCollectionTest() {
+        super(new InMemoryCollectionFactory(new Properties()));
+    }
 
+    @Test

```

Comment:
@cl-exense Shouldn't we have this test in AbstractCollectionTest ?

Reply:
Moved there, you'll have to reapprove the PR because it was changed :)